README.md

```diff
@@ -38,229 +88,412 @@
   one project and the projects that depend on it will automatically know they
   need to rebuild themselves.
 
-- Run commands in project environment quickly `esy any command`.
+- Run commands in project environment quickly `esy <anycommand>`.
 
 - Makes sharing of native projects easier than ever by supporting "eject to `Makefile`".
+
   - Build dependency graph without network access.
-  - Build dependency graph where `node` is not installed and where no package manager
-    is installed.
+
+  - Build dependency graph where `node` is not installed and where no package
+    manager is installed.
 
 ## Install
 
 ```
-npm install -g esy
+% npm install --global esy
 ```
 
 If you had installed esy previously:
 
 ```
-npm uninstall -g esy
-npm install -g esy
+% npm uninstall --global --update esy
 ```
 
 ## Workflow
 
-`esy` provides one global command (`esy`) which manages compiled, local
-`package.json` projects.
+Esy provides a single command called `esy`.
 
 The typical workflow is to `cd` into a directory that contains a `package.json`
 file, and then perform operations on that project.
 
-```
-% esy
+### Try An Example
 
-  Usage: esy <command> [--help] [--version]
+There are example projects:
```

Comment:
To anyone else reading this: We welcome other examples here! Send a PR. Another benefit of doing so, is that we can keep an eye on your project to send upgrade PRs to it when we improve stuff.